sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/README.md

```diff
@@ -1,13 +1,179 @@
-# Azure Opentelemetry Exporter for Monitor
+# Microsoft Opentelemtry Exporter for Azure Monitor client library for Python
```

Comment:
Please add gitter link 

[![Gitter chat](https://img.shields.io/gitter/room/Microsoft/azure-monitor-python)](https://gitter.im/Microsoft/azure-monitor-python)

Reply:
Actually I believe there is a gitter for Azure SDK, we can deprecate the old one and use the SDK one now
https://gitter.im/Azure/azure-sdk-for-python